tests/panels/UpdatePanelTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+class UpdatePanelTest extends FunctionalTest
+{
+    protected static $fixture_file = 'dashboard/tests/panels/UpdatePanelTest.yml';
+
+    public function testPermission()
+    {
+        $updatePanel = UpdatePanel::create();
+
+        $permittedUser = $this->objFromFixture('Member', 'user1');
+        $this->logInAs($permittedUser);
+        $this->assertTrue($updatePanel->canView());
+
+        $nonPermittedUser = $this->objFromFixture('Member', 'user2');
+        $this->logInAs($nonPermittedUser);
+        $this->assertNotTrue($updatePanel->canView());
```

Comment:
Do you want to change this to `assertFalse` instead? That is what CMS and Framework use in their tests.